private/gen/modules/state/v1alpha1/state.pb.go

```diff
@@ -100,8 +101,8 @@ func (b0 GlobalState_builder) Build() *GlobalState {
 // reference from its local array of references.
 type GlobalStateReference struct {
 	state                      protoimpl.MessageState `protogen:"opaque.v1"`
-	xxx_hidden_ModuleName      string                 `protobuf:"bytes,1,opt,name=module_name,json=moduleName,proto3" json:"module_name,omitempty"`
-	xxx_hidden_LatestReference string                 `protobuf:"bytes,2,opt,name=latest_reference,json=latestReference,proto3" json:"latest_reference,omitempty"`
+	xxx_hidden_ModuleName      string                 `protobuf:"bytes,1,opt,name=module_name,json=moduleName,proto3"`
```

Comment:
@unmultimedio - I don't think this will affect consumers but just FYI.

Reply:
No it won't, it's a missing piece of work they had from the opaque API: https://github.com/golang/protobuf/issues/1658